src/backend/routers/_app.ts

```diff
@@ -1,16 +1,16 @@
 import { router } from "../trpc";
-import { fileProcedures } from "./files";
-import { adminProcedures } from "./admins";
-import { paraProcedures } from "./paras";
-import { studentProcedures } from "./students";
-import { userProcedures } from "./users";
+import { files } from "./files";
+import { admin } from "./admins";
+import { paras } from "./paras";
+import { students } from "./students";
+import { users } from "./users";
 
 export const trpcRouter = router({
-  ...fileProcedures,
-  ...adminProcedures,
-  ...studentProcedures,
-  ...userProcedures,
-  ...paraProcedures,
+  files,
```

Comment:
minor: Consider using the singular form. Currently, admin is the only singular one in the list, while others are plural. Singular namespaces read more naturally, e.g. `file.getPresignedUrlForFileUpload` (can be later renamed to `file.getPresignedUrlForUpload`)

Reply:
agreed, minor brain fart